### ISSUE TITLE
Build error 

### ISSUE THREAD

**Issue by dheeraj-jft** [JimmyDaddy/react-native-image-marker#183]
I am getting this error while building the project 

Problems reading data from Binary store in /home/jellyfish/.gradle/.tmp/gradle6304675005526889684.bin offset 4195046 exists? true
Corrupt serialized resolution result. Cannot find selected module (334) for constraint releaseVariantReleaseRuntimePublication -> androidx.lifecycle:lifecycle-process:2.6.2

**Comment by JimmyDaddy** [comment]
@dheeraj-jft What is your gradle version? Is it below 8.3 ?

**Comment by JimmyDaddy** [comment]
> Yes, It is 8.0.1. Should I upgrade it?

Yes, [`Coil` ](https://github.com/coil-kt/coil) need gradle 8.0.3. [You can refer to the Gradle version settings in the example project](https://github.com/JimmyDaddy/react-native-image-marker/blob/69f58f79c98626e8102b30cec277e34d2937a002/example/android/gradle/wrapper/gradle-wrapper.properties#L3C5-L3C5).
